pyscriptjs/src/remote_interpreter.ts

```diff
@@ -262,4 +263,23 @@ export class RemoteInterpreter extends Object {
         const importlib = this.interface.pyimport('importlib') as PyProxy & { invalidate_caches(): void };
         importlib.invalidate_caches();
     }
+
+    pyimport(mod_name: string): PyProxy {
+        return this.interface.pyimport(mod_name);
+    }
+
+    mkdirTree(path: string) {
+        this.interface.FS.mkdirTree(path);
+    }
+
+    writeFile(path: string, content: string) {
+        this.interface.FS.writeFile(path, content, { encoding: 'utf8' });
+    }
+
+    async setHandler(func_name: any, handler: any): Promise<void> {
+        const pyscript_module = this.interface.pyimport('pyscript');
+        pyscript_module[func_name] = async (...args: any[]) => {
+            await handler(...args); // wrapper ensures task is scheduled
+        };
+    }
```

Comment:
This code is wrong and I think it's currently not used.

Reply:
Yeah, do you suggest removing this?
I mean, We added it to be used with `await interpreter._remote.setHandler`
but we switched back to `await interpreter._unwrapped_remote.setHandler` for now

Reply:
Yes. It's not good to commit dead code.

Reply:
yes, if it's not used it should not be committed

Reply:
Actually, it is used here: https://github.com/pyscript/pyscript/pull/1258/files#diff-a46bdf2eb41054c052aba50dd29116d21635c57e69de9b2596015f9670ab7a8dR276